### ISSUE TITLE
[BUG] Colvis button wont show in datatable that styled with tabler.

### ISSUE THREAD

**Issue by rsyafdhil** [tabler/tabler#1928]
### Browser

Chrome v22

### OS

Windows 11

### Screen size

1920x1800

### Describe the bug

did tabler hide the colvis button? cuz when i was styling my website with tabler, the colvis button wont showing again. Before i styled it with tabler, the colvis button showing up. Is that a bug? or tabler just doesnt showing the colvis button

### How to reproduce

1. add dom: 'Bfrtip' inside ur datatable script

### Screenshots

![image](https://github.com/user-attachments/assets/8c74478d-9d93-44c8-a0eb-fb602e4bb6eb)
![image](https://github.com/user-attachments/assets/e783d3eb-7e1a-43c5-b933-c61eb04a182f)

**Comment by FullStack-webDevloper** [comment]
Use Custom CSS to Override Conflicts:
You can write custom CSS to ensure that the ColVis button is visible and styled correctly. For example:


.dt-button.buttons-colvis {
  display: inline-block !important;
  visibility: visible !important;
  /* Add any additional styling needed to make it fit with Tabler's design */
}

**Comment by rsyafdhil** [comment]
thank you for answering my question, but it still doesnt work / showing to the view. i've check on the element source too but it still wont show.

**Comment by rsyafdhil** [comment]
until now i still dont know how to solve this bug, and i just make a script for column visibility itself. 

here's my code

|| column visibility dropdown
`
<div class="dropdown mb-3">
                            <button class="btn btn-secondary dropdown-toggle" type="button" data-bs-toggle="dropdown"
                                aria-expanded="false">
                                Column Visible
                            </button>
                            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton" id="columnVisibilityDropdown">
                                {{-- Dropdown items / th will be generated from script --}}
                            </div>
                        </div>
`

|| get the dropdown script
`
var columnVisibility = {};
$('#datatable thead th').each(function(index) {
                var columnName = $(this).text();
                var checkmark =
                    '<svg  xmlns="http://www.w3.org/2000/svg"  width="24"  height="24"  viewBox="0 0 24 24"  fill="none"  stroke="currentColor"  stroke-width="2"  stroke-linecap="round"  stroke-linejoin="round"  class="icon icon-tabler icons-tabler-outline icon-tabler-check"><path stroke="none" d="M0 0h24v24H0z" fill="none"/><path d="M5 12l5 5l10 -10" /></svg>'
                $('#columnVisibilityDropdown').append(
                    '<a class="dropdown-item toggle-vis" href="#" data-column="' + index + '">' +
                    columnName + checkmark + '</a>'
                );
                // Initialize all dropdown items as visible
                columnVisibility[index] = true;
            });
`

|| hide or show scripts
`
$('.toggle-vis').on('click', function(e) {
                e.preventDefault();
                var column = $(this).attr('data-column');
                var table = $('#datatable').find('tr');
                table.each(function() {
                    $(this).find('th, td').eq(column).toggle();
                });
                // Update visibility state
                columnVisibility[column] = !columnVisibility[column];
                if (columnVisibility[column]) {
                    $(this).find('svg').css('display', 'inline-block');
                } else {
                    $(this).find('svg').css('display', 'none');
                }
            });
`

yall can use this when ur colvis button still not showing.

**Comment by ac-mmi** [comment]
nothing related to Tabler
have you included cdn for button and colvis
```
<script src="https://cdn.datatables.net/buttons/3.1.0/js/dataTables.buttons.js"></script>
<script src="https://cdn.datatables.net/buttons/3.1.0/js/buttons.dataTables.js"></script>
<script src="https://cdn.datatables.net/buttons/3.1.0/js/buttons.colVis.min.js"></script>
```
mine is showing the colvis button

**Comment by rsyafdhil** [comment]
@ac-mmi yup, i already tried it but still wont showing the colvis button

**Comment by ac-mmi** [comment]
@rsyafdhil  would require a link
